packages/suite/src/views/wallet/send/index.tsx

```diff
@@ -3,7 +3,7 @@ import { ReactNode } from 'react';
 import styled from 'styled-components';
 
 import { Banner, Column } from '@trezor/components';
-import { breakpointMediaQueries } from '@trezor/styles';
+import { SCREEN_QUERY } from '@trezor/components/src/config/variables';
```

Comment:
nit: I suggest to import just the package:

```tsx
import { variables } from '@trezor/components';

...

${variables.SCREEN_QUERY.ABOVE_DESKTOP} {
```

or export it explicitly in `@trezor/components`

Reply:
Also thinking about refactoring naming and structure without keeping in mind our previous state. 
It would be really nice have everything on the same place and not import it from 3 different packages with different naming conventions:
- `SCREEN_QUERY` from `@trezor/components` 
- `useLayoutSize` from `@trezor/hooks`
- `breakpoints` from `@trezor/theme`

Not sure if we are able to unify all of these or useLayoutSize must be somewhere else. In ideal world I would suggest to do something like this:

```tsx
import { media } from '@trezor/???'; 

// media object
{
  query: {
    mobile: `@media ...`,
    aboveMobile: `@media ...`,
    ...
  }
  breakpoints: {
    mobile: 576,
    table: 768,
    ...
  }
  useQuery: fn
}
```

WDYT?